packages/libs/react-ui/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,38 @@
+import { tableClass } from './Table.css';
+import { TBody } from './TBody';
+import { THead } from './THead';
+import { Tr } from './Tr';
+import { CompoundType } from './types';
+
+import React, { FC } from 'react';
+
+export interface ITable {
+  children?: CompoundType<typeof TBody> | CompoundType<typeof THead>;
+}
+
+interface ITableComposition extends FC<ITable> {
+  Body: typeof TBody;
+  Head: typeof THead;
+  Tr: typeof Tr;
+}
+
+// eslint-disable-next-line react/prop-types
```

Comment:
What is this reason for this? We want to avoid disabling lint errors when possible